contents/node/nodejs.md

```diff
@@ -103,14 +115,14 @@ A basic `package.json` has the following structure.
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "keywords": [],
+  "keywords": ["promise", "lock"],
   "author": "",
   "license": "ISC"
 }
 ```
-Every package needs a **name** and a **version**. Together, they should form a unique identifier. When a package is updated, the version number must be updated. A good **description** string and **keywords** e.g. `["promise", "lock"]` will help others to discover your package.
+The **name** and a **version** forms a unique identifier for the package. When a package is updated, the version number must be updated. A good **description** string and **keywords** helps others in discovering your package.
```

Comment:
Consider changing "helps others in discovering..." to "helps others discover..."